unify directives presentation (#211)
neo4j/docs-graphql: modules/ROOT/pages/security/subscriptions-authorization.adoc

= Subscriptions authorization

Subscriptions require their own authorization rules, which are configured with the `@subscriptionsAuthorization` directive.

== `@subscriptionsAuthorization`

> Rules for subscriptions authorization are different to normal authorization rules because only filtering rules are available for subscriptions events and subscriptions events have more limitations in how they can be filtered.

Reviewer: ```suggestion
Rules for subscriptions authorization are different to normal authorization rules.
Only filtering rules are available for subscriptions events and there are more limitations in how they can be filtered.
```

All subscriptions authorization rules have an implied requirement for authentication, given that the rules are normally evaluated against values in the JWT payload.

> === Definition
> 
> // tba

Reviewer 2: ```suggestion
```

Same as others.

```
[source, graphql, indent=0]
----
"""
This is a simpler version of the subscriptionsAuthorization directive to be used in the validate-document step.
"""
directive @subscriptionsAuthorization(
  """filter"""
  filter: [String]!
) on OBJECT | FIELD_DEFINITION
----
```
